MastodonSDK/Sources/MastodonCore/Extension/CoreDataStack/Status+Property.swift

```diff
@@ -51,14 +51,28 @@ extension Mastodon.Entity.Status {
         guard let mediaAttachments = mediaAttachments else { return [] }
         
         let attachments = mediaAttachments.compactMap { media -> MastodonAttachment? in
-            guard let kind = media.attachmentKind,
-                  let meta = media.meta,
-                  let original = meta.original,
-                  let width = original.width,       // audio has width/height
-                  let height = original.height
+            guard let kind = media.attachmentKind
             else { return nil }
-            
-            let durationMS: Int? = original.duration.flatMap { Int($0 * 1000) }
+
+            let width: Int;
+            let height: Int;
+            let durationMS: Int?;
+
+            if let meta = media.meta,
+               let original = meta.original,
+               let originalWidth = original.width,
+               let originalHeight = original.height {
+                width = originalWidth;              // audio has width/height
+                height = originalHeight;
+                durationMS = original.duration.flatMap { Int($0 * 1000) }
```

Comment:
I’m pretty sure that the `Int` initializer you’re using here is non-bailable, so you can use a regular `map`. Let me know if this is wrong though :​)

```suggestion
                durationMS = original.duration.map { Int($0 * 1000) }
```